kafka-connect/kafka-connect/src/test/java/org/apache/iceberg/connect/data/IcebergWriterFactoryTest.java

```diff
@@ -83,4 +90,26 @@ public void testAutoCreateTable(boolean partitioned) {
     assertThat(specCaptor.getValue().isPartitioned()).isEqualTo(partitioned);
     assertThat(propsCaptor.getValue()).containsKey("test-prop");
   }
+
+  @Test
+  public void testNamespaceCreation() throws IOException {
```

Comment:
This is unit test. In future when the integration tests are added, we can add the auto table creation with the missing namespaces integration tests.